.github/workflows/e2e_ci.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         with:
           pixi-version: v0.34.0
           cache: true
-          cache-write: ${{ github.event_name == 'push' && github.ref_name == 'main' }}
+          cache-write: ${{ github.event_name == 'schedule' }}
```

Comment:
I think it was just wrong originally. Before, it would save when the CI runs on pushed commits to the main branch. But the CI doesn't trigger in that case, so it never saves the cache. Now, it saves the cache on every nightly run.